notify/discord/discord.go

```diff
@@ -90,14 +97,20 @@ func (n *Notifier) Notify(ctx context.Context, as ...*types.Alert) (bool, error)
 		return false, err
 	}
 
-	title := tmpl(n.conf.Title)
+	title, truncated := notify.TruncateInRunes(tmpl(n.conf.Title), maxTitleLenRunes)
 	if err != nil {
 		return false, err
 	}
-	description := tmpl(n.conf.Message)
+	if truncated {
+		level.Warn(n.logger).Log("msg", "Truncated title", "incident", key, "max_runes", maxTitleLenRunes)
```

Comment:
```suggestion
		level.Warn(n.logger).Log("msg", "Truncated title", "key", key, "max_runes", maxTitleLenRunes)
```

I believe we call this `key` in other notifiers e.g. https://github.com/prometheus/alertmanager/blob/main/notify/slack/slack.go#L116C55-L116C58

Reply:
Fixed in beecbcb